src/Integrations/Talentlyft/SettingsTalentlyft.php

```diff
@@ -0,0 +1,293 @@
+	 *
+	 * @return array<int, array<string, mixed>>
+	 */
+	public function getSettingsGlobalData(): array
+	{
+		// Bailout if feature is not active.
+		if (!UtilsSettingsHelper::isOptionCheckboxChecked(self::SETTINGS_TALENTLYFT_USE_KEY, self::SETTINGS_TALENTLYFT_USE_KEY)) {
+			return UtilsSettingsOutputHelper::getNoActiveFeature();
+		}
+
+		$deactivateIntegration = UtilsSettingsHelper::isOptionCheckboxChecked(self::SETTINGS_TALENTLYFT_SKIP_INTEGRATION_KEY, self::SETTINGS_TALENTLYFT_SKIP_INTEGRATION_KEY);
+
+		return [
+			UtilsSettingsOutputHelper::getIntro(self::SETTINGS_TYPE_KEY),
+			[
+				'component' => 'tabs',
+				'tabsContent' => [
+					[
+						'component' => 'tab',
+						'tabLabel' => \__('General', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'checkboxes',
+								'checkboxesFieldLabel' => '',
+								'checkboxesName' => UtilsSettingsHelper::getOptionName(self::SETTINGS_TALENTLYFT_SKIP_INTEGRATION_KEY),
+								'checkboxesContent' => [
+									[
+										'component' => 'checkbox',
+										'checkboxLabel' => UtilsSettingsOutputHelper::getPartialDeactivatedIntegration('checkboxLabel'),
+										'checkboxHelp' => UtilsSettingsOutputHelper::getPartialDeactivatedIntegration('checkboxHelp'),
+										'checkboxIsChecked' => $deactivateIntegration,
+										'checkboxValue' => self::SETTINGS_TALENTLYFT_SKIP_INTEGRATION_KEY,
+										'checkboxSingleSubmit' => true,
+										'checkboxAsToggle' => true,
+									]
+								]
+							],
+							...($deactivateIntegration ? [
+								[
+									'component' => 'intro',
+									'introSubtitle' => UtilsSettingsOutputHelper::getPartialDeactivatedIntegration('introSubtitle'),
+									'introIsHighlighted' => true,
+									'introIsHighlightedImportant' => true,
+								],
+							] : [
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => true,
+								],
+								UtilsSettingsOutputHelper::getPasswordFieldWithGlobalVariable(
+									UtilsSettingsHelper::getSettingsDisabledOutputWithDebugFilter(
+										Variables::getApiKeyTalentlyft(),
+										self::SETTINGS_TALENTLYFT_API_KEY_KEY,
+										'ES_API_KEY_TALENTLYFT'
+									),
+									\__('API key', 'eightshift-forms'),
+								),
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => true,
+								],
+								UtilsSettingsOutputHelper::getTestAliConnection(self::SETTINGS_TYPE_KEY),
+							]),
+						],
+					],
+					[
+						'component' => 'tab',
+						'tabLabel' => \__('Options', 'eightshift-forms'),
+						'tabContent' => [
+							...$this->getGlobalGeneralSettings(self::SETTINGS_TYPE_KEY),
+							[
+								'component' => 'input',
+								'inputName' => UtilsSettingsHelper::getOptionName(self::SETTINGS_TALENTLYFT_FILE_UPLOAD_LIMIT_KEY),
+								'inputFieldLabel' => \__('Max upload file size', 'eightshift-forms'),
+								'inputFieldHelp' => \__('Up to 5MB.', 'eightshift-forms'),
```

Comment:
Can the value of `self::SETTINGS_TALENTLYFT_FILE_UPLOAD_LIMIT_KEY` be `sprint_f`...ed here? 😄